nsv13/code/game/gamemodes/pvp/base_defense.dm

```diff
@@ -0,0 +1,225 @@
+	icon_state = "shuttlered"
+
+/area/tartarus/exterior/surfacebase
+	name = "Bowie base 2 access"
+	looping_ambience = 'sound/weather/ashstorm/inside/active_mid1.ogg'
+
+/area/tartarus/interrogation
+	name = "Bowie base 2 interrogation"
+	looping_ambience = 'nsv13/sound/ambience/medbay.ogg'
+	icon_state = "medbay"
+
+/area/tartarus/base
+	name = "Bowie base 2"
+
+/area/tartarus/tcomms
+	name = "Bowie base 2 TE/LE/COMM core"
+	looping_ambience = 'nsv13/sound/ambience/computer_core.ogg'
+	icon_state = "tcomsatcham"
+
+/area/tartarus/commodore
+	name = "Bowie base 2 commodore's office"
+	looping_ambience = 'nsv13/sound/ambience/bridge.ogg'
+	icon_state = "syndie-control"
+
+/area/tartarus/briefing
+	name = "Bowie base 2 briefing room"
+	looping_ambience = 'nsv13/sound/ambience/bridge.ogg'
+	icon_state = "syndie-control"
+
+/area/tartarus/armoury
+	name = "Bowie base 2 armoury"
+	icon_state = "syndie-elite"
+
+/area/tartarus/maintenance
+	name = "Bowie base 2 maintenance ducts"
+	looping_ambience = 'nsv13/sound/ambience/maintenance.ogg'
+	icon_state = "maintcentral"
+
+/area/tartarus/lounge
+	name = "Bowie base 2 lounge"
+
+/area/tartarus/meeting
+	name = "Bowie base 2 meeting room"
+
+/area/tartarus/cryostasis
+	name = "Bowie base 2 cryostasis"
+
+/area/tartarus/engineering
+	name = "Bowie base 2 reactor core"
+
+/obj/structure/overmap/planet/pvp
+	name = "Tartarus"
+	desc = "A desolate frozen wasteland that seems resistant to surface scans..."
+	icon = 'nsv13/icons/overmap/stellarbodies/planets.dmi'
+	icon_state = "planet_cloud"
+	mass = MASS_IMMOBILE
+	damage_states = FALSE
+	collision_positions = list(new /datum/vector2d(-7,110), new /datum/vector2d(-44,98), new /datum/vector2d(-76,67), new /datum/vector2d(-94,26), new /datum/vector2d(-89,-19), new /datum/vector2d(-69,-59), new /datum/vector2d(-33,-86), new /datum/vector2d(-6,-95), new /datum/vector2d(42,-92), new /datum/vector2d(82,-63), new /datum/vector2d(99,-28), new /datum/vector2d(105,3), new /datum/vector2d(91,57), new /datum/vector2d(62,91), new /datum/vector2d(29,104))
+	starting_system = "Scorvio"
+	obj_integrity = INFINITY
+	max_integrity = INFINITY //Yeah. Because. Planets don't just...up and die...or go into SS crit. Y'know?
+	armor = list("overmap_light" = 100, "overmap_heavy" = 80) //Good luck with war-criming a planet Czanek.
+	pixel_z = -128
+	pixel_w = -128
+	faction = "syndicate"
+	ai_controlled = FALSE
+	role = PVP_SHIP
+
+//Overridable method which spawns in the ship we want to use.
+/datum/game_mode/pvp/base_defense/generate_ship(ship_type, map_file)
+	ship_type = /obj/structure/overmap/planet/pvp //Yes. I'm a hack. Sue me Fran.
+	return instance_overmap(_path=ship_type, folder= "map_files/PVP" ,interior_map_files = map_file, traits=list(list(ZTRAIT_UP=1, ZTRAIT_LINKAGE = SELFLOOPING, ZTRAIT_BOARDABLE = 1),list(ZTRAIT_DOWN = -1, ZTRAIT_LINKAGE = SELFLOOPING, ZTRAIT_BOARDABLE = 1)))
+
+
+/datum/game_mode/pvp/base_defense/force_lighting(obj/structure/overmap/hammurabi)
```

Comment:
Fucking hell. There has to be *some* way to do this that doesn't involve a waitfor-false proc and *multiple sleeps*